.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+# FIXME: master -> main
+on:
+  push:
+    branches: 
+        - 'master'
+        - 'RELEASE-*'
```

Comment:
But this project does not use branches of such pattern. It's rather `*-RELEASE`.

Reply:
@pzygielo - good catch.  I'll update that once the master -> main change happens. 
The other Jakarta repos I've worked on had it reversed so I didn't notice at first.